flask_blogging/engine.py

```diff
@@ -154,3 +156,17 @@ def process_post(self, post, render=True):
     def get_user_name(cls, user):
         user_name = user.get_name() if hasattr(user, "get_name") else str(user)
         return user_name
+
```

Comment:
This should perhaps be called something like `static_asset_loader` because you seem to be loading more than bootstrap. 

Also there was a reason why I had markdown and bootstrap markdown load on the editor view. That way you are not loading unnecessary static assets for all the other pages. Google will ding sites like this with unoptimised static assets being loaded.